README.md

```diff
@@ -153,29 +155,30 @@ You can also use a [global configuration](#global-configuration) to provide defa
 - [Suspense Mode](#suspense-mode)
 - [Error Retries](#error-retries)
 - [Prefetching Data](#prefetching-data)
+- [Request Deduplication](#request-deduplication)
 
 ### Global Configuration
 
-The context `SWRConfig` can provide global configurations (`options`) for all SWR hooks. 
+The context `SWRConfig` can provide global configurations (`options`) for all SWR hooks.
 
 In this example, all SWRs will use the same fetcher provided to load JSON data, and refresh every 3 seconds by default:
 
 ```js
 import useSWR, { SWRConfig } from 'swr'
 
-function Dashboard () {
+function Dashboard() {
   const { data: events } = useSWR('/api/events')
   const { data: projects } = useSWR('/api/projects')
   const { data: user } = useSWR('/api/user', { refreshInterval: 0 }) // don't refresh
   // ...
 }
 
-function App () {
+function App() {
   return (
-    <SWRConfig 
+    <SWRConfig
       value={{
         refreshInterval: 3000,
-        fetcher: (...args) => fetch(...args).then(res => res.json())
+        fetcher: (...args) => fetch(...args).then((res) => res.json()),
```

Comment:
Thank you for doing code linting! However we don't have a preferred style in our contribution guide, so I'd keep things like these unedited. Also it is for the better consistency with the [documentation site](https://swr.vercel.app).

Reply:
@shuding agreed! I have reverted the changes that affected the code examples. Minor spaces and indentations are kept.